README.md

```diff
@@ -63,3 +88,41 @@ Developers on a OS for which there is no binary package, or who would like to st
 1. Run `make`.
 
 This will build the fdbserver binary and the python bindings. If you want to build our other bindings, you will need to install a runtime for the language whose binding you want to build. Each binding has an `.mk` file which provides specific targets for that binding.
+
+#### Windows (Experimental)
```

Comment:
I think this needs to be marked as CMake for windows somehow, because clearly we can build and produce windows installers (via the old build system).  I'd recommend either moving this section up or the CMake section down, and maybe use a heading difference to clearly split between what instructions are for CMake and what instructions are for vcxproj.

Reply:
I disagree: I was unable to build on Windows with the old build system. It doesn't work out-of the box and some paths to dependencies are hard-coded

Reply:
Hm, I might be splitting hairs, but that seems less "experimental" and more "it was bad".

I would still recommend doing the shuffle to keep the CMake parts next to each other, and emphasize that there are two different build systems.  If we end up with a header structure of:

```
## Makefile
### Linux
### MacOS

## CMake
### Linux
### Windows
```

Then that's fine, and we'll fill in the MacOS later, at least before removing `Makefile` and all its per-platform instructions

Reply:
I can change the ordering sure - but I still disagree about the Windows build ;) I would call it generally non-existent with the caveat that there is a machine somewhere that can build it and I am sure if you're willing to invest a day or two you might be able to compile it (but I was not able to do it within a few hours).

But it doesn't really matter. I think this will need to be updated anyways as soon as the docker-file can compile with cmake
But it doesn't really matter. CMake will make the bul